person.rb

```diff
@@ -30,6 +31,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
- [ ] You should create an entirely new instance of rentals here. It can go like `Rental.new(date, book, self)` but you should remember to add other parameters to the initializer of the rental class.